### ISSUE TITLE
`gtest` does not run correctly under macOS with GitHub Actions

### ISSUE THREAD

**Issue by zaikunzhang** [equipez/PRIMA#9]
[`gtest`](https://github.com/equipez/PRIMA/actions/runs/3665412852/workflow) fails with the following message when testing macOS on GitHub Actions. 

```
ld: library not found for -lgcrt1.o
collect2: error: ld returned 1 exit status
```
It used to work, but now it fails, with the `gfortran` version being 9, 10, 11, or, 12.

N.B.: 
1. Despite the failure, the test did not terminate with an error but continued. This is not really ideal. Probably the [makefile](https://github.com/equipez/PRIMA/blob/master/fsrc/tests/makefiles/Makefile.common) needs revision.
2. In the GitHub Actions tests, the `gfortran` is provided by [awvwgk / setup-fortran](https://github.com/awvwgk/setup-fortran).

**Comment by zaikunzhang** [comment]
`gtest`  fails with the following message when testing macOS on GitHub Actions. 

```
ld: library not found for -lgcrt1.o
collect2: error: ld returned 1 exit status
```
It used to work, but now it fails, with the `gfortran` version being 9, 10, 11, or, 12.

N.B.: 
1. Despite the failure, the test did not terminate with an error but continued. This is not really ideal. Probably the [makefile](https://github.com/equipez/PRIMA/blob/master/fsrc/tests/makefiles/Makefile.common) needs revision.
2. In the GitHub Actions tests, the `gfortran` is provided by [awvwgk / setup-fortran](https://github.com/awvwgk/setup-fortran).

**Comment by zaikunzhang** [comment]
Solved by [fbe58d1](https://github.com/equipez/PRIMA/commit/fbe58d154fd93c1d97919ca8b98220e7dc235228). It was because `gfortran` was invoked with `-pg`, which was used to compile for profiling with `gprof` in a previous test. This option necessitates` gcrt1.o`.  Everything works after `-pg` is removed.
